tests/console/snapper_undochange.pm

```diff
@@ -15,22 +23,24 @@ use testapi;
 sub run() {
     select_console 'root-console';
 
-    my $snapfile = '/root/snapfile';
+    # snapper undochange [options] number1..number2 [files] return -> numb snapshot created
 
-    my $snapbf = script_output "snapper create -p -d 'before undochange test'";
-    script_run "date > $snapfile";
-    my $snapaf = script_output "snapper create -p -d 'after undochange test'";
+    # create the 2 snapshots, inbetweeen create the file. then go back when file doesn't exist
+    type_string('before_snap=$(snapper create -p -d \'before undochange test\');');
+    type_string('snapfile="/root/test_snapfile";date > $snapfile; after_snap=$(($before_snap + 1));');
+    assert_script_run('test -f $snapfile');
+    type_string('snapper create -p -d \'after undochange test\';');
+    type_string('snapper undochange $before_snap..$after_snap $snapfile;');
+    type_string('snapper list; ls /root;');
```

Comment:
are you sure this will succeed in making your tests more robust?

The issue described in https://progress.opensuse.org/issues/10386 suggests that default script_output timeout was unstable

Instead you've replaced it with type_strings which have absolutely no checking and zero delay between the commands

The problems I expect if we merge this would be as follows

We'd have no idea if your `snapper create` worked

If `snapper create` takes more than a few miliseconds, openQA will start typing the `snapper undochange` while the snapper create is still running

and everything I just said would also happen again with the `snapper list` on line 34

The screenshot you've provided doesn't give me much confidence as it's not even showing the test module in question - a direct link to your local openQA test run might give me some reassurance that the above problem won't occur..but I need convincing

If we need to increase the default timeout of script_output from 10 seconds to something like 30 seconds, I think that should be a pull request against os-autoinst and we shouldn't be working around the less-than-ideal API calls with what I fear is an even worse approach